daemon/Cargo.toml

```diff
@@ -6,6 +6,7 @@ authors = ["Radicle Team <dev@radicle.xyz>"]
 edition = "2018"
 
 [dependencies]
+anyhow = "1.0.40"
```

Comment:
Does it have to be specific to the patch version? Or could we use `"1"`?

Reply:
@FintanH I had misunderstood how this works in the cargo world. This being "^1.0.40", and cargo being mucho semantico, the meaning is ">= 1.0.40 && <2.0.0". If the first digit was a zero, it would mean ">= 0.0.40 && <0.0.41".

Reply:
(please do not ask "what about the digit in the middle?")

Reply:
Dios mio :scream: I think you're right. `1.0.40` includes the hidden `^` and this is the table they list:
```
^1.2.3  :=  >=1.2.3, <2.0.0
^1.2    :=  >=1.2.0, <2.0.0
^1      :=  >=1.0.0, <2.0.0
^0.2.3  :=  >=0.2.3, <0.3.0
^0.2    :=  >=0.2.0, <0.3.0
^0.0.3  :=  >=0.0.3, <0.0.4
^0.0    :=  >=0.0.0, <0.1.0
^0      :=  >=0.0.0, <1.0.0
```

I really don't understand this... So _any_ non-backward compatible change in `>=1` has to be a major change rather than minor.

So this is saying that we need changes from `1.0.40` and above but nothing below. Is that the expectation @alexjg?

Reply:
> I really don't understand this... So any non-backward compatible change in >=1 has to be a major change rather than minor.

Ah wait, I've confused myself...

Reply:
No, no, that's correct, as you can convince yourself of here: https://semver.org/lang/ar/#%D8%A8%D8%A7%D8%AE%D8%AA%D8%B5%D8%A7%D8%B1

Haskellers have the luxury of an additional digit in the version specifier, but it's not clear if that reduces the total amount of confusion

Reply:
I've changed it to `"1.0.0"`. There is an argument that maybe we shouldn't depend on `anyhow` at all as it's generally intended for applications rather than libraries. But `daemon` is more like an application anyway so I think it's okay.

Reply:
FWIW the rust "error community" considers `anyhow` deprecated and advises alternatives like `eyre`.

Reply:
<sup>[citation needed]</sup>

Reply:
I sufficiently bamboozled myself and mistook the `anyhow` crate for the `anomaly` crate - which is being sunset. In the process of digging for citation I found this: https://github.com/rust-lang/project-error-handling